variable/technology/power-plant.yaml

```diff
@@ -0,0 +1,756 @@
+# List of variables related to power plants
+# Variables related to the production cost of a power plant
+# Note that:
+# linear   term cost   [€/GWh] = Linear Var Cost [Mcal/GWh] * Price|Primary Energy [€/Mcal] + Operation and Maintenance Variable Cost [€/MWh] * 1e-3
+# constant term cost   [€/h]   = Constant Var Cost [Mcal/h] * Price|Primary Energy [€/Mcal]
+Constant Var Cost|Electricity|Coal:
+   description: Constant term (intercept) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category. It refers to the unit commitment variable of each power plant per unit time.
+   unit: Mcal/h
+
+Constant Var Cost|Electricity|Gas:
+   description: Constant term (intercept) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category. It refers to the unit commitment variable of each power plant per unit time.
+   unit: Mcal/h
+
+Constant Var Cost|Electricity|Nuclear:
+   description: Constant term (intercept) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category. It refers to the unit commitment variable of each power plant per unit time.
+   unit: Mcal/h
+
+Constant Var Cost|Electricity|Oil:
+   description: Constant term (intercept) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category. It refers to the unit commitment variable of each power plant per unit time.
+   unit: Mcal/h
+
+Linear Var Cost|Electricity|Coal:
+   description: Linear term (slope) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category. It refers to the production cost of each energy unit (GWh, MWh,etc.).
+   unit: Mcal/GWh
+
+Linear Var Cost|Electricity|Gas:
+   description: Linear term (slope) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category. It refers to the production cost of each energy unit (GWh, MWh,etc.).
+   unit: Mcal/GWh
+
+Linear Var Cost|Electricity|Nuclear:
+   description: Linear term (slope) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category. It refers to the production cost of each energy unit (GWh, MWh,etc.).
+   unit: Mcal/GWh
+
+Linear Var Cost|Electricity|Oil:
+   description: Linear term (slope) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category. It refers to the production cost of each energy unit (GWh, MWh,etc.).
+   unit: Mcal/GWh
+
+Operation and Maintenance Cost|Electricity|Coal:
```

Comment:
I am not sure to understand what Operation and Maintenance cost means. If it is the cost per MWh when the plant is generating power I do not understand why call it maintenance cost? For me, maintenance cost is in the fixed costs....
More generally, I think we should discuss a bot more all the 'costs' section. I also added some variables called "Variable Cost" which in my modelling are only the linear term of the function (unit €/MWh) (we also have quadratic cost and fixed cost in the model but as our public dataset has no values for fixed and quadratic terms I did not add the variables). this "variable cost" is meant to comprise every component of the linear cost (can be human cost, fuel cost/or price, anything in fact).